src/interfaces/network.ts

```diff
@@ -14,6 +14,10 @@ interface FeeOptions {
   // transactions on Base get stuck on slow as we lower the baseFee a lot
   // so we make the minBaseFee the same as the last block one
   minBaseFeeEqualToLastBlock?: boolean
+  userOpFees?: {
+    // all the fees are in percentages
```

Comment:
why is this required in FeeOptions? is it different for each fee option or fixed?

also, using promilles or basis points is more appropriate with BigInts since it allows for more precision

Reply:
This is removed as discussed offline